py/htm/advanced/examples/thalamus/README.md

```diff
@@ -0,0 +1,28 @@
+
+
+
+An implementation of thalamic control and routing as proposed in the Cosyne submission:
+
+    A dendritic mechanism for dynamic routing and control in the thalamus Carmen Varela & Subutai Ahmad
+
+Data
+====
+
+The data is a single image `cajal.jpg` 
+
+Filters
+=======
+
+To avoid adding [scikit-image](https://scikit-image.org/) as a requirement for running this example, a local copy of the gabor filter kernel is provided.
```

Comment:
rather do use it from pip. Either figure the "extras" option for `requirenments.txt` (we'd also add matplotlib), or just doc here to `pip install skimage`. If the example is run, used as a test, feel free to install skimage by default.

Reply:
ok,I see, it pulls quite some deps. thoughts? I'd still just pip it, probably.